ja/routing.md

```diff
@@ -47,21 +47,21 @@ export function createApp () {
 // entry-server.js
 import { createApp } from './app'
 export default context => {
-  // since there could potentially be asynchronous route hooks or components,
-  // we will be returning a Promise so that the server can wait until
-  // everything is ready before rendering.
+  // 非同期のルートフックまたはコンポーネントが存在する可能性があるため、
+  // プロミスを返却します。
```

Comment:
ここですが、原文的には、`we will be returning a Promise so that the server can wait until everything is ready before rendering.` だと思いますので、訳は以下になるのではないでしょうか。

```
レンダリングする前にすべての準備が整うまでサーバーが待機できるようにプロミスを返します。
```